elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metric/EntityMetric.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.datastores.aggregation.metric;
+
+import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.datastores.aggregation.annotation.MetricAggregation;
+
+import java.util.List;
+
+/**
+ * Default implementation of {@link Metric} that represents an entity field as a metric.
+ * <p>
+ * {@link EntityMetric} is thread-safe and can be accessed by multiple threads.
+ */
+public class EntityMetric extends AbstractMetric {
+
+    private static final long serialVersionUID = -3848782355816135844L;
+
+    /**
+     * Constructs a new {@link EntityMetric} instance.
+     *
+     * @param metricField  The entity field of the metric being constructed
+     * @param cls  The entity that contains the metric being constructed
+     * @param entityDictionary  The auxiliary object that offers binding info used to construct this
+     * {@link EntityMetric}
+     *
+     * @return a {@link Metric}
+     */
+    public static Metric newInstance(String metricField, Class<?> cls, EntityDictionary entityDictionary) {
```

Comment:
Same comment - keep all of these construction methods in the schema or find a way to not propagate the EntityDictionary so widely.